include/multitasking.h

```diff
@@ -9,3 +9,7 @@
  * run.
  */
 void yield();
+
+int mutex_lock_yield(int *lock);
```

Comment:
and some docs on these

Reply:
agree.  They'll get added in a later commit I hope